__tests__/app.test.js

```diff
@@ -157,6 +158,26 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200: when a topic query is requested, returns a filtered array of the articles with the requested topic", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles.length).toBe(12);
+        articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
```

Comment:
We're just missing one more test here Rosie. What would happen if it was a valid topic but has no articles?